From: DropDemBits <r3usrlnd@gmail.com>
Subject: mtd/spi-nor/xmc: add support for XMC XM25QH256B

The XMC XM25QH256B is a 32MB SPI NOR chip, which is used in some
Deco M5 v3 and v320 models.

Datasheet is available at:
https://www.xmcwh.com/uploads/424/XM25QH(QU)256B_Ver.G.pdf

Signed-off-by: DropDemBits <r3usrlnd@gmail.com>
---
 drivers/mtd/spi-nor/xmc.c                             | 3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/mtd/spi-nor/xmc.c
+++ b/drivers/mtd/spi-nor/xmc.c
@@ -22,6 +22,8 @@ static const struct flash_info xmc_nor_p
 	{ "XM25QH128C", INFO(0x204018, 0, 64 * 1024, 256,
			     SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
+	{ "XM25QH256B", INFO(0x206019, 0, 64 * 1024, 512,
+			     SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
 };
 
 const struct spi_nor_manufacturer spi_nor_xmc = {
